src/request/src/request.js

```diff
@@ -177,6 +180,7 @@ export default class Request {
 
   toPlainObject() {
     return {
+      id: this.id,
```

Comment:
This `id` is generated by the SDK and used to match up log statements for requests. Multiple requests might be logged at the same time and without some kind of identifier it would be hard to match log statements that are related. We need to communicate that this is not related to the `X-Kinvey-Request-Id` header by adding the information to the troubleshooting guide.